torch/distributed/_composable/fsdp/fully_shard.py

```diff
@@ -23,6 +23,9 @@
 from ._fsdp_state import _get_module_fsdp_state, FSDPState
 
 
+cls_to_fsdp_wrapper: Dict[Type, Type] = {}
```

Comment:
nit: wrapper might be a tiny bit misleading
```suggestion
cls_to_fsdp_cls: Dict[Type, Type] = {}
```